### ISSUE TITLE
Volume disk crypted with VeraCrypt, detected as Bitlocker

### ISSUE THREAD

**Issue by slow8** [veracrypt/VeraCrypt#524]
Hi,

I have a situation regarding the name of the subject. Last Friday I was upgrading my old computer to an new one. I had a latitude 5470 and now i own a 5400. I've done a fresh install of the latest windows os, and created 2 patitions (C and D) and another one that was a shrink from D, called E - 3 GB - for the encryption purpose. 
So far so good...after i was able to format and encrypt the disk, I transferred all my sensitive infos from my old notebook to new one over the network. Ejected the volume and put the computer on sleep mode. 
Here comes the problem..After a restart or 2, the encrypted partition(E) was recognized as a bitlocker partition and when i try to open it, always displays the following message : 

~ The Bitlocker encryption on this drive isn't compatible with the version of windows. Try opening the drive using newer a version of windows~ 

But it is the newest version of windows 10 1903.

Can you please help me solve this problem, couse all of my important data dare locked in there...

Thanks

**Comment by idrassi** [comment]
Are you able to mount the encrypted partition E in VeraCrypt? If not, what error do you get form VeraCrypt when you do so?

**Comment by slow8** [comment]
At first, till the restart, the pictogram from E, was looking like a bitlocker encyption, and the i ve tryed to unlock it with VeraCrypt, the error was that the password is incorect. but is was always correct.

And now, when I try to mount E with VeraCrypt, it says that the drive is locked by BitLocker Encryption and i must unlock the drive from control panel. **Source : MountVolume:8292**

**Comment by idrassi** [comment]
We can’t do much if Bitlocker is taking over the drive and I don’t know why it is doing so. It is the first time such issue is reported and normally partitions encrypted by VeraCrypt are seen as RAW by the system and Bitlocker don’t touch them since they don’t look formatted.

In your place, the first thing I will do is to try mount this partition outside of Windows, for example by booting the machine on a Linux Live USB key. This will indicate if the partition is still recognizable by VeraCrypt or if it has been overwritten by Bitlocker.
Then, if it mounting with VeraCrypt on Linux, I would change its password in order to force the creation of a new volume header value: this volume header has a random value but maybe in your case this random value matches bytes sequence expected by Bitlocker (this is very highly unlikely but it is not impossible.

**Comment by aastolbov** [comment]
Hello!
I want to say that I have an identical case. And your advice helped me, thanks!

**Comment by slow8** [comment]
For me, does not work...I've tried the solution with the Linux usb live, but no success. 

Can you please tell me what was in your case ?

Thanks

**Comment by aastolbov** [comment]
My system disk has 2 partitions, Windows installed on the first, second is a VeraCrypt container (Winows shows it as RAW partition). I turned on Bitlocker on system Windows drive, and after reboot VeraCrypt partition has become displayed as an encrypted bitlocker drive. 
It became impossible to mount it through VeraCrypt, I got "Source : MountVolume:8292" error.
I booted my PC via LiveUSB with Windows 10, started VeraCrypt Portable and successfully mount my VeraCrypt partition. Then I changed passowrd of conainer, and now can mount it in Windows, it shown as RAW partition
